changelogs/fragments/1152-lambda_info-bugfix-all-lambdas.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - lambda_info - fix bug that forces query=config when getting info for all lambdas. Now, if function name is specified, query will default to all.
```

Comment:
```suggestion
  - lambda_info - fix bug that forces query=config when getting info for all lambdas. Now, if function name is specified, query will default to all.  This may have a performance impact when querying a large number of lambdas.
```